Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -485,7 +485,7 @@ def PyExec(self):  # noqa
             # END-FOR
         # ENDIF (Sum data or not)
 
-        for (samRunIndex, sam_ws_name) in enumerate(samwksplist):
+        for (samExpInfoIndex, sam_ws_name) in enumerate(samwksplist):
```

Comment:
All changes should be reverted in this file. `samRunIndex` is unrelated to `MDEvent.runIndex`

Reply:
done!